site/templates/application/footer/nav.jet

```diff
@@ -1,13 +1,16 @@
+{{import "./social-media-icons.jet"}}
+
 {{ block footerNav(navLinks) }}
   {{ if len(navLinks) > 0 }}
-    <nav class="footer-nav" aria-label="{{ i18n("wcag_aria_label_footer") }}">
+    <nav class="footer-nav" aria-label='{{ i18n("wcag_aria_label_footer") }}'>
         <ul class="nav">
           {{ range navLinks }}
-            <li class="nav-item {{ len(.Link.ExternalURL) == 0 && routeToSlug(.Link.Slug) == currentUrlPath ? "active" : "" }}">
+            <li class='nav-item {{ len(.Link.ExternalURL) == 0 && routeToSlug(.Link.Slug) == currentUrlPath ? "active" : "" }}'>
               <a class="nav-link" href="{{ len(.Link.ExternalURL) > 0 ? .Link.ExternalURL : routeToSlug(.Link.Slug) }}">{{ .Label }}</a>
             </li>
           {{ end }}
         </ul>
+        {{ yield footerSocialMediaIcons() }}
```

Comment:
This isn't a good place to put this... if they have no nav-links in the footer-nav, then the social media buttons wont render either.

Reply:
I've sorted this by moving the nav out of the if statement and setting margin-left to auto for the social links div